kermit-bugsnag/README.md

```diff
@@ -70,9 +70,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
-## Dynamic Frameworks
+## Testing
+
+If you're building a dynamic framework, or you're using Kotlin version 1.8.0+ which builds dynamic by default,
```

Comment:
The change in 1.8.0 was that cocoapods used to default to static, and now defaults to dynamic. But I think if you weren't using cocoapods it was previously defaulting to dynamic and there's no change. That nuance might not matter for these docs though